app/redux/wallet/actions.js

```diff
@@ -73,6 +73,19 @@ export const saveNewWallet = ({ mnemonic, salt = cryptoConsts.DEFAULT_SALT }: {
   }
 };
 
+export const saveCurrentWallet = (): Action => (dispatch: Dispatch, getState: GetState): Dispatch => {
+  const walletState = getState().wallet;
+  const { wallet, fileKey } = walletState;
+  const encryptedAccountsData = cryptoService.encryptData({ data: JSON.stringify(wallet.crypto.cipherText), key: fileKey });
+  const encryptedWallet = { ...wallet, crypto: { cipher: 'AES-128-CTR', cipherText: encryptedAccountsData } };
+  try {
+    fileSystemService.saveFile({ fileContent: JSON.stringify(encryptedWallet), showDialog: true });
+    dispatch(updateWalletData({ wallet }));
```

Comment:
why calling wallet data update? No change happened during backup. Remove it

Reply:
ok